inc/fonts.php

```diff
@@ -33,10 +23,11 @@ function bootstrap() {
  * @param array $defaults The default settings for the font directory.
  */
 function pantheon_font_dir( $defaults ) {
-	global $_pantheon_upload_dir;
+	$upload_dir = wp_get_upload_dir();
+
 	// Set our font directory.
-	$font_dir = $_pantheon_upload_dir['basedir'] . '/fonts';
-	$font_url = $_pantheon_upload_dir['baseurl'] . '/fonts';
+	$font_dir = $upload_dir['basedir'] . '/fonts';
+	$font_url = $upload_dir['baseurl'] . '/fonts';
 
```

Comment:
Can't make a suggestion on the whole function with the structure of the diff,— I took a stab at simplifying but I'm swinging back and forth over here whether the following makes it more or less readable. Apply if you wish. 
```
function pantheon_font_dir( $defaults ) {
	$upload_dir = wp_get_upload_dir();

	// Set our font directory.
	$defaults['basedir'] = $upload_dir['basedir'] . '/fonts';
	$defaults['baseurl'] = $upload_dir['baseurl'] . '/fonts';

	$defaults['path'] = $defaults['basedir'];
	$defaults['url'] = $defaults['baseurl'];
	return $defaults;
}
```

Reply:
I feel like defining `$font_dir`/`$font_url` and then saving those to the various `$defaults` values is more explicit about what's going on.